### PR TITLE
solution

### PR DESCRIPTION
**Pull request by NikitaKarbazin**
[DEMO LINK](https://nikitakarbazin.github.io/react_person/)

### CODE REVIEW

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,36 @@
...
+  name,
+  partnerName,
+  sex,
+}) => {
+  let personPartner = 'I am not married';
+
+  if (partnerName) {
+    personPartner = sex === 'f' ? `My husband's name is ${partnerName}`
+      : `My wife's name is ${partnerName}`;
+  }
```
Comment (stanislav-topikha) [inline src/components/Person/Person.jsx:15]: Kinda weird logic if you already have this prop: `isMarried`
What if the person have a partner, but they are not married(gf/bf)?

File: src/App.js [hunk 2]
```diff
@@ -28,22 +28,9 @@ const alex = {
...
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    <Person {...misha} />
+    <Person {...olya} />
+    <Person {...alex} />
   </div>
```
Comment (stanislav-topikha) [inline src/App.js:34]: 👍

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
Great 👍  but ⬇️

File: src/components/Person/Person.jsx (hunk 1 above)
Comment (NikitaKarbazin) [reply src/components/Person/Person.jsx:15]: ahahha! It's my wierd logic! I want to find bicycle)) But smth went wrong

**Review by stanislav-topikha — APPROVED** [review]

File: src/components/Person/Person.jsx [hunk 3]
```diff
@@ -1 +1,39 @@
...
+        {' '}
+        {name}
+      </h2>
+      {age && (
+        <p className="Person__age">
+          I am
+          {' '}
+          {age}
+        </p>
+      )}
```
Comment (Olexandr6) [inline src/components/Person/Person.jsx:33]: [Checklist](https://github.com/mate-academy/react_sum/blob/master/checklist.md) the 3rd point

**Review by Olexandr6 — APPROVED** [review]

**Pull request closed without merging**